src/journal/journald-server.c

```diff
@@ -1573,6 +1574,36 @@ static int server_parse_proc_cmdline(Server *s) {
                                 log_warning("Failed to parse forward to wall switch %s. Ignoring.", word + 33);
                         else
                                 s->forward_to_wall = r;
+                } else if (startswith(word, "systemd.journald.max_level_console=")) {
+                        r = log_level_from_string(word + 35);
+                        if (r < 0)
+                                log_warning("Failed to parse max level console value %s. Ignoring.", word + 35);
```

Comment:
Hm, I know that this is the style that is prevalently used, but I really dislike those manual numbers like this 35. What about a follow up patch to do this:
```
const char *val;
…
else if ((val = startswith(word, "systemd.journald.max_level_console="))) {
        r = log_level_from_string(val);
        ...
```
?